src/content/docs/apm/agents/manage-apm-agents/app-naming/use-multiple-names-app.mdx

```diff
@@ -15,36 +15,40 @@ redirects:
   - /docs/apm/new-relic-apm/installation-configuration/use-multiple-names-app
 ---
 
-import apmInsightsToggleonandOff from 'images/apm_screenshot-full_insights-toggle-on-and-off.png'
+When working with your APM data in New Relic, you may want to view aggregated, "rolled up" data for an application that spans multiple clusters, environments, or data centers, but also be able to view each of the app instance's data separately. You can do this by setting multiple app names for your APM-monitored apps.
 
-When working with your data in New Relic, you may want to view aggregated data for an application across clusters, environments, or data centers, while at the same time be able to view each of your application instance's data individually.
+## Important notes [#important-notes]
 
-<Callout variant="important">
-  If you use multiple names for an app, the last name assigned is the least specific name New Relic uses to roll up the data. For best results to view both instance-level and aggregated data, set the order of your app's rollup names from **most specific to least specific** in your configuration file.
-</Callout>
+Here are some important caveats to be aware of when using multiple app names.
 
-## Roll up app data [#rollup]
+### Alternative options [#alt-options]
 
-Normally, when two instances report with the same app name, agent language, and [license key](/docs/apis/intro-apis/new-relic-api-keys/#ingest-license-key), New Relic aggregates their data into a single New Relic-monitored app. You can also view data for the individual hosts and instances from the app's New Relic [APM **Overview** page](/docs/apm/applications-menu/monitoring/apm-overview-page#infra-server).
+Because adding multiple app names results in [duplicate data](#duplicate), you may want to consider these other options. 
 
-To segment your app in a different way, or to work with each instance as an individual app while maintaining the overall view of the data, use your New Relic agent's [config file](#agent) to give an app up to three different rollup names. New Relic will then report the app's data separately to each app listed in the config file.
+If your goal is to be able to filter your telemetry data by attributes (for example, like the agent region), we recommend one of these options instead:
 
-For example, you might want to separate the data collected for your app running in development, staging, and production environments, but also have a common view of the app in every environment. Or, if you have two data centers running the same code, you could name one app `EastCoastApp;AggregateApp` and the second app `WestCoastApp;AggregateApp`.
+* Add tags via the agent config file (for example, [this Java config](/docs/apm/agents/java-agent/configuration/java-agent-configuration-config-file/#labels))
+* Use [custom attributes](/docs/data-apis/custom-data/custom-events/collect-custom-attributes/#enabling-custom)
```

Comment:
@zuluecho9 Probably worth saying a little more about what custom attributes are used for. As it stands, I'm not sure I would have much sense about why I'd use this instead of tags, say